cli/src/pcluster/aws/aws_resources.py

```diff
@@ -224,10 +224,13 @@ def instance_type(self):
         return self.instance_type_data.get("InstanceType")
 
     def is_cpu_options_supported_in_lt(self):
```

Comment:
since you have changed the method behavior, please also check all the callers of this method, to understand if they need to be changed as well.

Reply:
Thanks, done in the next rev.